recipe/meta.yaml

```diff
@@ -119,7 +185,6 @@ about:
     - LICENSE_CUTLASS.txt
 
 extra:
-  feedstock_name: {{ name|lower }}
```

Comment:
Looks like this got dropped accidentally. We will want to keep this so the feedstock name is still set correctly in the README, etc. (given the top-level package name changes)

Reply:
Done.